### PR TITLE
Unit Tests for plus Button Functionality and Various other tasks

### LINKED ISSUE #35: Add Button Unit Testing
- creating two tests. one for adding the first movie title in the list, the second for the second movie title.
- these tests are related to the add button '+'.
- testing that it picks up the input/button of the plus/add button.
- check if it clicks with the event handler.
- checking that the list container will pick up or simply contain the movie-title.

### PR DESCRIPTION
**Pull request by PJSalter**
Sorry About the Branch Name it's a little misleading 😆 🤖 

- 🎬 I have created two unit tests for the adding a title button, check them to see if they are up to scratch, let me know cause they pass, however i'm not sure if there passing correctly 😆 
- 🎬 checking the var for the inputBtn
- 🎬 checking the movieTitle.
- 🎬 clicking the '+' button to enter the title of the movie within the binge list
- 🎬 checking if they add the first and second element title.
- 🎬 changed movie-title class colours to the CSS Variables 
- 🎬 Also this PR contains where I am at with the edit functionality, I thought I'll show what I have so far, I haven't got it to fully work as of yet, I think it's super close 🤔 , if you both have any advise or would like to look at it then your welcome to check it out 💭 ✨ 

Hope this all makes sense i'm a little tired 😆 

📺 🍿 🍬 📼 🎟️ 🍿 🍫 📺

### CODE REVIEW

File: index-test.js [hunk 1]
```diff
@@ -16,6 +16,82 @@ test("Deleting an entry removes it from the list", () => {
...
+  const inputTitle = document.getElementsByClassName('list-input')
+   // giving the input a value otherwise new movie will be created
+  inputTitle.value = "Show me that new movie";
+  // what happens after the click.
+  const expected = inputTitle.childElementCount;
+  //checking the click of an item
+  // then user should then click the button.
+  //checking the click of an item
+  //adding a catcher at the beginning of the function seemed to fix the error of cannot read property of undefined.
+  if(this.item === undefined) {return}
```
Comment (AsmahanM) [inline index-test.js:84]: This was super helpful! Nice to see all your comments too, really clear structure.

**Review by AsmahanM — APPROVED** [review]

File: index-test.js [hunk 2]
```diff
@@ -55,6 +55,82 @@ setTimeout(() => {
...
```
Comment (mariaalouisaa) [inline index-test.js:84]: Really nice comments throughout!

File: index-test.js [hunk 3]
```diff
@@ -55,6 +55,82 @@ setTimeout(() => {
...
+  inputTitle.childElementCount;
+  //cheking the class attribute for the button
+  let bingeWorthy = document.querySelector(".movie-name");
+  //console.log(plusTitleToLife)
+   // giving the input a value otherwise new movie will be created
+  inputTitle.value = "yay to second choice";
+  // what happens after the click.
+  const expected = inputTitle.childElementCount;
+
+  equal(result, expected + 2);
```
Comment (mariaalouisaa) [inline index-test.js:121]: You've clearly worked really hard on this! Great job 🥳🙌

File: style.css [hunk 4]
```diff
@@ -1,345 +1,341 @@
...
+    position: fixed;
+    z-index: 10;
+    background-color: transparent;
+    border: none;
+    height: 10em;
   }
+  
   .tv img {
-  width: 7em;
+    width: 10em;
```
Comment (mariaalouisaa) [inline style.css:97]: It' seems likes all the CSS changes are just the responsive branch merging with your comp? I'm going to merge and will double check the deploy after :)

File: index-test.js [hunk 5]
```diff
@@ -55,6 +55,82 @@ setTimeout(() => {
...
+  //to fix the error of inputBtn not being defined
+  const inputBtn = document.querySelector('button')
+  //InputDiv.childElementCount
+   //picks up the input button on click
+  const inputTitle  = document.querySelector(".list-input")
+  //const result = document.getElementsByClassName('input-list')
+  // then user should then click the button.
+  //checking the click of an item
+  //adding a catcher at the beginning of the function seemed to fix the error of cannot read property of undefined.
+  if(this.item === undefined) {return}
```
Comment (mariaalouisaa) [inline index-test.js:109]: This is interesting! I'm going to have a play around with it after the pull in my own console to understand it more :) Really enjoyed reading your code too!!

**Review by mariaalouisaa — APPROVED** [review]
Awesome work 🙌

**Pull request merged**